### PR TITLE
Updated SKR provision fast-integration step

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Updated SKR provison fast-integration step

**Related issue(s)**

### CODE REVIEW

File: tests/fast-integration/skr-test/provision-skr.js [hunk 1]
```diff
@@ -30,13 +30,13 @@ describe('Provision SKR instance', function() {
   });
 
   after('Print Shoot Info', async function() {
-    if (shootInfo != undefined) {
-      console.log(options);
+    // print data out for spinnaker
+    if (options && options.instanceID) {
+      console.log(`SPINNAKER_PROPERTY_INSTANCE_ID=${options.instanceID}`);
```
Comment (tillknuesting) [inline tests/fast-integration/skr-test/provision-skr.js:35]:
```suggestion
      console.log(`INSTANCE_ID=${options.instanceID}`);
```

File: tests/fast-integration/skr-test/provision-skr.js [hunk 2]
```diff
@@ -30,13 +30,13 @@ describe('Provision SKR instance', function() {
...
+    }
 
-      // print data out for spinnaker
-      console.log('***Print out data for spinnaker***');
+    if (shootInfo) {
       console.log(`SPINNAKER_PROPERTY_PROVISIONED=true`);
```
Comment (tillknuesting) [inline tests/fast-integration/skr-test/provision-skr.js:39]:
```suggestion
      console.log(`PROPERTY_PROVISIONED=true`);
```

File: tests/fast-integration/skr-test/provision-skr.js (hunk 1 above)
Comment (tillknuesting) [inline tests/fast-integration/skr-test/provision-skr.js:35]:
```suggestion
      console.log(`PROPERTY_INSTANCE_ID=${options.instanceID}`);
```

Comment (mfaizanse) [reply tests/fast-integration/skr-test/provision-skr.js:35]: It needs to have the prefix "SPINNAKER_PROPERTY_" as required by spinnaker.

File: tests/fast-integration/skr-test/provision-skr.js (hunk 2 above)
Comment (mfaizanse) [reply tests/fast-integration/skr-test/provision-skr.js:39]: It needs to have the prefix "SPINNAKER_PROPERTY_" as required by spinnaker.

File: tests/fast-integration/skr-test/provision-skr.js (hunk 1 above)
Comment (tillknuesting) [reply tests/fast-integration/skr-test/provision-skr.js:35]: could you elaborate on it why?

File: tests/fast-integration/skr-test/provision-skr.js (hunk 2 above)
Comment (tillknuesting) [reply tests/fast-integration/skr-test/provision-skr.js:39]: could you elaborate on it why?

Comment (mfaizanse) [reply tests/fast-integration/skr-test/provision-skr.js:39]: It used in spinnaker provision stage to pass data to next stage. More info: https://spinnaker.io/docs/guides/user/kubernetes-v2/run-job-manifest/#spinnaker_property_

**Review by tillknuesting — APPROVED** [review]

**Pull request merged**